### PR TITLE
Nit: Fix listing formats for inline docs.

### PR DESCRIPTION
**Pull request by sunoru**
Fix some inline docs that contain incorrectly-formatted lists. For example, docs for `x` in [`predict_on_batch`](https://www.tensorflow.org/api_docs/python/tf/keras/Model?hl=en#predict_on_batch).

### CODE REVIEW

**Review by bhack — APPROVED** [review]
Let see Kokoro response

**Review by tomerk — APPROVED** [review]
Thanks for the cleanup!

**Comment by bhack** [comment]
@tomerk do you know why `ready to pull` label will trigger another build without commit?
